### PR TITLE
Support true sequential options in gptq

### PR DESCRIPTION
**Pull request by YIYANGCAI**
## Type of Change

Feature

### CODE REVIEW

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py [hunk 1]
```diff
@@ -333,7 +340,7 @@ def eval_func(model):
         eval_func=eval_func,
     )
 
-    q_model.save(args.output_dir)
+    # q_model.save(args.output_dir)
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py:343]: why remove save?

**Comment by chensuyue** [comment]
@YIYANGCAI do we still need this PR?

**Review by xin3he — APPROVED** [review]
LGTM, I will raise another PR to support 3x.

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py (hunk 1 above)
Comment (YIYANGCAI) [reply examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py]: remove for test. Now "save" has been restored.

**Review by changwangss — APPROVED** [review]

**Comment by changwangss** [comment]
ITREX PR https://github.com/intel/intel-extension-for-transformers/pull/1559 apply the feature

File: examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py [hunk 2]
```diff
@@ -358,7 +361,6 @@ def eval_func(model):
             tokenizer=tokenizer,
             tasks=args.tasks,
             batch_size=args.batch_size,
-            args=args,
```
Comment (chensuyue) [inline examples/pytorch/nlp/huggingface_models/language-modeling/quantization/llm/run_clm_no_trainer.py:361]: @changwangss can we remove this one?

**Pull request merged**